### PR TITLE
Feature/year month navigation

### LINKED ISSUE #33: Improve navigation using year and month in routes
The proposal here is to make navigation in the application simpler.

For this, we can use year and month in the routes. The idea is that the routes have the following behavior:

- `https://celebrityfanalyzer.web.app/2022`: return all prompts of 2022;
- `https://celebrityfanalyzer.web.app/2022/november`: return the prompt of November and its entries.

In my opinion, the ideal is not to create just an _alias_, but rather a **slug** change for both prompts and entries.

### PR DESCRIPTION
**Pull request by hermannleboss**
Using id instead of using slug in the prompt page routing

image

Close #33

### CODE REVIEW

**Comment by arnonrdp** [comment]
Hey @hermannleboss, aliases didn't work well?
Isn't it possible to use the same component for `MonthArchivePage.vue` and `PromptPage.vue`?

**Comment by hermannleboss** [comment]
Explain

**Comment by arnonrdp** [comment]
> Explain

Please check: https://router.vuejs.org/guide/essentials/redirect-and-alias.html#alias

We use aliases so that the user can access the same destination with different URLs.

Furthermore, it is also possible to reuse `PrompPage.vue` as the Component to be rendered for this URL.

In the end it would be something like:
```js
{
   path: 'prompt/:id',
   alias: ':year(\\d{4})/:month',
   component: () => import('pages/PromptPage.vue')
}
```

This way, the user will be able to access the page through `year/month` or `prompt/slug`.
I even believe that for sharing it is better that we use the slug, but that is another point.

File: src/components/ItemCard.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <q-avatar size="2rem">
           <q-img :src="item.author.photoURL" />
         </q-avatar>
-        <p class="q-mb-none q-ml-sm text-body1">
+        <p class="q-mb-none q-ml-sm text-body1" v-if="item.author.displayName">
```
Comment (arnonrdp) [inline src/components/ItemCard.vue:8]: I already fixed the bug we were having here.

You can `git pull` it from `develop` branch

**Review by arnonrdp — CHANGES_REQUESTED** [review]
Please check: https://router.vuejs.org/guide/essentials/redirect-and-alias.html#alias

We use aliases so that the user can access the same destination with different URLs.

Furthermore, it is also possible to reuse `PrompPage.vue` as the Component to be rendered for this URL.

In the end it would be something like:
```js
{
   path: 'prompt/:id',
   alias: ':year(\\d{4})/:month',
   component: () => import('pages/PromptPage.vue')
}
```

This way, the user will be able to access the page through `year/month` or `prompt/slug`.
I even believe that for sharing it is better that we use the slug, but that is another point.

**Comment by hermannleboss** [comment]
OK i see

**Comment by hermannleboss** [comment]
It's OK, but i have an issue. 
The question is what parameter do we receive from the routing. 

```
   path: 'prompt/:id',
   alias: ':year(\\d{4})/:month',
```

We have `id` in `prompt/:id` and `year` and `month` in `year(\\d{4})/:month`

And with this parameter we should make the request to the store to find the right one.

**Review by arnonrdp — APPROVED** [review]

**Pull request merged**